src/dos/dos_locale.cpp

```diff
@@ -1348,6 +1732,86 @@ static const std::map<Country, CountryInfo> CountryData = {
 			Separator::Semicolon      // list separator
 		} }
 	} } },
+	{ Country::Oman, { "Oman", "OMN", {
+		{ LocalePeriod::Modern, {
+			// (taken from the common Arabic, adapted the currency)
```

Comment:
Locale should be ar_OM
https://sourceware.org/git/?p=glibc.git;a=blob;f=localedata/locales/ar_OM;h=863084f089b2a2690e0b3496cc7e9f27c5f10bd1;hb=HEAD